fuse/plugins/micro_physics/yields.py

```diff
@@ -87,13 +83,14 @@ def get_quanta(self, interactions_in_roi):
             interactions_in_roi["A"],
             interactions_in_roi["Z"],
             interactions_in_roi["create_S2"],
+            log=self.log,
             density=interactions_in_roi["xe_density"],
         )
 
         return photons, electrons, excitons
 
     @staticmethod
-    def _quanta_from_NEST(en, model, e_field, A, Z, create_s2, **kwargs):
+    def _quanta_from_NEST(en, model, e_field, A, Z, create_s2, log, **kwargs):
```

Comment:
this will change with PR #245 , but in principle it is good. (this function will not be static anymore, so self.log will be good)

Reply:
Hi @cfuselli you are right, I will update this part when resolving the merge conflicts of the linked PR.